### PR TITLE
Remove repetitive test cases

### PR DESCRIPTION
**Pull request by sbulage**
Fixes: #49

### CODE REVIEW

**Review by mmartin24 — APPROVED** [review]
LGTM

File: tests/cypress/e2e/unit_tests/first_login_rancher.spec.ts [hunk 1]
```diff
@@ -13,14 +12,11 @@ limitations under the License.
...
 
 Cypress.config();
 describe('First login on Rancher', () => {
-  qase(46,
     it('Log in and accept terms and conditions', () => {
     cypressLib.firstLogin();
     })
-  );
-})
+})
```
Comment (sbulage) [inline tests/cypress/e2e/unit_tests/first_login_rancher.spec.ts:22]: I need to add new line at the end of this file, I will add it after QASE integration PR gets in.

**Review by thehejik — APPROVED** [review]
LGTM

**Review by mmartin24 — APPROVED** [review]
LGTM

**Pull request merged**